system_test/common/aest_channels_SUITE.erl

```diff
@@ -292,8 +293,8 @@ reestablish_state_channel_perform_operations_close({INodeName, RNodeName},
     wait_for_value({txs_on_chain, [CloseTxHash]}, NodeNames, 5000, Config),
     %% wait for min depth to be reached so the channels die
     #{height := TopHeight} = aest_nodes:get_top(INodeName),
-    KeyBlocksToMine = 4 + 2, % min depth is 4
-    wait_for_value({height, TopHeight + KeyBlocksToMine}, NodeNames, 10000, Config),
+    KeyBlocksToMine = ?MIN_DEPTH + 2, % min depth is 4
```

Comment:
So mining 13 blocks instead of 6? that makes the test slower, can it be avoided?

Reply:
Returned old value

Reply:
Since we verify that the min depth strategy works fine with other values, we could use a factor of `0` here and in general in other tests, which would require only a single block.